src/templates/tr/2019/contributors.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base/contributors.html" %}
+
+{% block title %}2019 Katkıda bulunanlar | HTTP Archive&#8217;dan Web Almanac{% endblock %}
+
+{% block description %}2019 Web Almanac&#8217;a Analistler, Yazarlar, Beyin Takımı, Tasarımcılar, Geliştiriciler, Editörler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }} kişi.{% endblock %}
```

Comment:
Meta data should be pain text as it's not rendered as HTML so would prefer not to use smart quotes like `&#8217` if we can avoid it.

```suggestion
{% block title %}2019 Katkıda bulunanlar | HTTP Archive'dan Web Almanac{% endblock %}

{% block description %}2019 Web Almanac'a Analistler, Yazarlar, Beyin Takımı, Tasarımcılar, Geliştiriciler, Editörler, İnceleyiciler ve Çevirmenler olarak katkıda bulunan {{ config.contributors.items() | length }} kişi.{% endblock %}
```